src/filter/index.js

```diff
@@ -15,16 +15,14 @@ export default async function filter(content, selector, filterNames, filterFunct
     });
   }
 
-  if (selector) {
-    const selectedContent = webPageDOM.querySelector(selector);
+  let selectedContent = [ webPageDOM ];
 
-    if (selectedContent) {
-      contentToFilter = selectedContent;
-    } else {
-      console.warn(`The provided selector "${selector}" has no match in the web page.`)
+  if (selector) {
```

Comment:
We agreed that, since this is validated in the schema, a `selector` will always be present. This test is thus irrelevant.

I'll take care of that change.